src/login/logind-session-dbus.c

```diff
@@ -901,6 +922,11 @@ static const sd_bus_vtable session_vtable[] = {
                                 SD_BUS_NO_RESULT,
                                 method_set_type,
                                 SD_BUS_VTABLE_UNPRIVILEGED),
+        SD_BUS_METHOD_WITH_ARGS("SetDisplay",
+                                SD_BUS_ARGS("s", type),
```

Comment:
Maybe it is better the method name more specific, e.g. `SetDisplayName`.
And the argument name should be `type` -> `name`.

Reply:
> Maybe it is better the method name more specific, e.g. `SetDisplayName`.

That would only cause confusion, as the corresponding (read-only) property for this is named `Display`.

> And the argument name should be `type` -> `name`.

Sorry, copypasta...